vita3k/modules/SceAppMgr/SceAppMgr.cpp

```diff
@@ -445,9 +445,24 @@ EXPORT(int, _sceAppMgrLoopBackMount) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, _sceAppMgrMmsMount) {
-    TRACY_FUNC(_sceAppMgrMmsMount);
-    return UNIMPLEMENTED();
+EXPORT(SceInt32, _sceAppMgrMmsMount, SceInt32 id, char *mount_point) {
+    TRACY_FUNC(_sceAppMgrMmsMount, id, mount_point);
+    switch (id) {
+    case 0x190:
+        strcpy(mount_point, "ux0:mms/photo/");
+        break;
+    case 0x191:
+        strcpy(mount_point, "ux0:mms/music");
+        break;
+    case 0x192:
+        strcpy(mount_point, "ux0:mms/video");
+        break;
+    default:
+        LOG_WARN("Unknown id: {}", log_hex(id));
```

Comment:
You should strcpy something here (like "ux0:mms/unkown" or whatever) if not returning an error

Reply:
Hmm no if is missing is probleme, and normally here i set all, so yes, must be adding error

Reply:
i have adding error